readme.txt

```diff
@@ -0,0 +1,38 @@
+=== Littlefoot ===
+Contributors: s3rgiosan, goblindegook
+Donate link: https://ko-fi.com/s3rgiosan
+Tags: footnotes, littlefoot, bigfoot, block
+Requires at least: 6.2
+Tested up to: 6.4
+Stable tag: 1.1.0
+Requires PHP: 7.4
+License: MIT
+License URI: https://opensource.org/license/mit/
+
+Elegant, interactive popups for the Footnotes block.
+
+== Description ==
+
+Elevate your website's user experience by incorporating lightweight and visually appealing footnotes using [littlefoot](https://github.com/goblindegook/littlefoot).
```

Comment:
@goblindegook I think that we need to make obvious to the user that this plugin doesn't provide a block and that works on top of the core's Footnotes block. 

Something like:
```
Elevate your website's user experience by incorporating lightweight and visually appealing footnotes using littlefoot. This WordPress plugin extends the WordPress Footnotes block functionality by leveraging littlefoot.
```

Reply:
@goblindegook have you had the time to review this?